### ISSUE TITLE
 Unhandled rejection ReferenceError: processAddress is not defined

### ISSUE THREAD

**Issue by Enase** [aws/aws-xray-sdk-node#52]
We are experiencing an issue with `node_modules/aws-xray-sdk-core/lib/daemon_config.js`

stack trace:
```
Unhandled rejection ReferenceError: processAddress is not defined
    at Object.<anonymous> (C:\****\node_modules\aws-xray-sdk-core\lib/daemon_config.js:24:17)
    at Module._compile (module.js:652:30)
    at Module._compile (C:\****\node_modules\pirates\lib\index.js:83:24)
    at Module._extensions..js (module.js:663:10)
    at Object.newLoader [as .js] (C:\****\node_modules\pirates\lib\index.js:88:7)
    at Module.load (module.js:565:32)
    at tryModuleLoad (module.js:505:12)
    at Function.Module._load (module.js:497:3)
```

OS: win
node: 8.11

**Comment by haotianw465** [comment]
Hi, thank you for reporting this issue. Do you have a short code snippet to reproduce the problem? I cannot reproduce this on unit test or my own sample app. It might be a missing keyword `var` at https://github.com/aws/aws-xray-sdk-node/blob/master/packages/core/lib/daemon_config.js#L24 but we want to be sure that you are having the same issue so we can provide the right fix. Please let me know.

**Comment by haotianw465** [comment]
I'm happy to hear this workaround works for you. We will fix this and roll it out soon.

**Comment by haotianw465** [comment]
This is out as part of `v2.0.1`.

**Comment by haotianw465** [comment]
This issue is fixed as part of `v2.1.0` release.
